File: ProjetoTCC/BancoDados/importPlanilhaExcel.vb
Imports System.Data.OleDb

Public Class importPlanilhaExcel

    Sub BuscarPlanilha(caminhoArquivoExcel As String, nomePlanilhaExcel As String)

        Dim cmd As New OleDbCommand
        nomePlanilhaExcel = nomePlanilhaExcel & "$"
        Dim conexaoOleDb As OleDbConnection = Nothing
        conexaoOleDb = New OleDbConnection("provider=Microsoft.Jet.OLEDB.4.0;Data Source= " & caminhoArquivoExcel & ";Extended Properties=Excel 8.0;")
        cmd.CommandText = "Select * from [" & nomePlanilhaExcel & "]"

    End Sub

End Class

File: ProjetoTCC/BancoDados/importPlanilhaExcel.cs
using System.Data.OleDb;

namespace ProjetoTCC
{
    public class importPlanilhaExcel
    {
        public void BuscarPlanilha(string caminhoArquivoExcel, string nomePlanilhaExcel)
        {
            var cmd = new OleDbCommand();
            nomePlanilhaExcel = nomePlanilhaExcel + "$";
            OleDbConnection conexaoOleDb = null;
            conexaoOleDb = new OleDbConnection("provider=Microsoft.Jet.OLEDB.4.0;Data Source= " + caminhoArquivoExcel + ";Extended Properties=Excel 8.0;");
            cmd.CommandText = "Select * from [" + nomePlanilhaExcel + "]";
        }
    }
}